elements/lisk-bft/src/finality_manager.ts

```diff
@@ -411,27 +411,17 @@ export class FinalityManager extends EventEmitter {
 			CONSENSUS_STATE_VALIDATOR_LEDGER_KEY,
 		);
 		const { ledger } = this._decodeVotingLedger(bftState);
-		const { preVoted } = this._getChainMaxHeightStatus(ledger);
-
-		return preVoted;
+		return this._calculateMaxHeightPrevoted(ledger);
 	}
 
-	private _getChainMaxHeightStatus(ledger: LedgerMap): { preVoted: number; preCommitted: number } {
+	private _calculateMaxHeightPrevoted(ledger: LedgerMap): number {
 		debug('updatePreVotedAndFinalizedHeight invoked');
 
 		const highestHeightPreVoted = Object.keys(ledger)
 			.reverse()
 			.find(key => ledger[key].preVotes >= this.preVoteThreshold);
 
-		const preVoted = highestHeightPreVoted ? parseInt(highestHeightPreVoted, 10) : 0;
-
-		const highestHeightPreCommitted = Object.keys(ledger)
-			.reverse()
-			.find(key => ledger[key].preCommits >= this.preCommitThreshold);
-
-		const preCommitted = highestHeightPreCommitted ? parseInt(highestHeightPreCommitted, 10) : 0;
-
-		return { preVoted, preCommitted };
+		return highestHeightPreVoted ? parseInt(highestHeightPreVoted, 10) : this.finalizedHeight;
```

Comment:
```suggestion
		return chainMaxHeightPreVoted ? parseInt(chainMaxHeightPreVoted, 10) : this.finalizedHeight;
```

Reply:
Updated to `maxHeightPrevoted` since thats the name we use everywhere else